spec/lib/dradis/plugins/projects/upload/v2/template_spec.rb

```diff
@@ -0,0 +1,38 @@
+require 'rails_helper'
+
+describe Dradis::Plugins::Projects::Upload::V2::Template::Importer do
+  let(:project) { create(:project) }
+  let(:user) { create(:user) }
+  let(:importer_class) { Dradis::Plugins::Projects::Upload::Template }
+  let(:file_path) do
+    File.join(
+      File.dirname(__FILE__),
+      '../../../../../../',
+      'fixtures',
+      'files',
+      'with_comments.xml'
+    )
+  end
+
+  context 'uploading a template with comments' do
+    before do
+      importer = importer_class::Importer.new(
+        default_user_id: user.id,
+        plugin: importer_class,
+        project_id: project.id
+      )
+
+      importer.import(file: file_path)
+    end
+
+    it 'imports comments in issues' do
```

Comment:
Same comment as above, we can save time if we combine these two `expect` so as to save time instead of importing twice.